configs/components/cpp-hocon.rb

```diff
@@ -11,7 +11,12 @@
   if platform.is_macos?
     toolchain = ""
     cmake = "/usr/local/bin/cmake"
-    special_flags = "-DCMAKE_CXX_FLAGS='#{settings[:cflags]}'"
+    boost_static_flag = "-DBOOST_STATIC=OFF"
```

Comment:
If we're turning off static boost, shouldn't we have to copy the `.dylib` files into our lib directory to ensure they'll be available on end-user systems?

Reply:
Added boost in puppet-runtime: [142](https://github.com/puppetlabs/puppet-runtime)